src/main/java/camp/nextstep/edu/nextstep8/racing/RacingGame.java

```diff
@@ -5,49 +5,44 @@
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class RacingGame {
     private static final String SEPARATOR = ",";
-
     private int roundTimes;
-    private List<RacingCar> entryList;
-    private List<RacingEntry> record = new ArrayList<>();
+    private RacingEntry entry;
 
     public RacingGame(String cars, int roundTimes) {
-        this.entryList = generateEntryList(cars);
+        this.entry = new RacingEntry(generateEntryList(cars));
         this.roundTimes = roundTimes;
     }
 
     public List<RacingEntry> raceStart() {
-        RacingEntry entry = new RacingEntry(entryList);
-        for(int i = 0; i < roundTimes; i++){
+        List<RacingEntry> record = new ArrayList<>();
+        for(int i = 0; i < roundTimes; i++) {
             entry.move(getRaceRule());
             record.add(getSnapShot(entry));
         }
 
-        return record;
+        return Collections.unmodifiableList(record);
+    }
+
+    public ForwardingRule getRaceRule() {
+        return new RandomForwardingRule();
     }
 
-    public List<RacingCar> generateEntryList(String cars) {
+    private List<RacingCar> generateEntryList(String cars) {
         return Arrays.stream(cars.split(SEPARATOR))
                 .map(RacingCar::new)
                 .collect(Collectors.toList());
     }
```

Comment:
```suggestion
    private RacingEntry generateEntryList(String cars) {
        return Arrays.stream(cars.split(SEPARATOR))
                .map(RacingCar::new)
                .collect(Collectors.collectingAndThen(Collectors.toList(), RacingEntry::new));
    }
```
위와 같이 사용할 수 있을것 같아요 !